.pre-commit-config.yaml

```diff
@@ -16,6 +16,7 @@ repos:
     hooks:
       # - id: fmt // Not currently supported for the nightly build, e.g., https://github.com/doublify/pre-commit-rust/issues/15
       - id: cargo-check
+  # Once this issue is resolved, we can add the full license check: https://github.com/Lucas-C/pre-commit-hooks/issues/68
```

Comment:
I thought we had gotten rid of these pre commit hooks?  They're pretty slow...

Reply:
No, they're still running :) This one is actually not that bad and it gives good signal (i.e., ensures all rust files have the header defined in `devtools/assets/license_header.txt`